docs/markdown/snippets/meson_options.md

```diff
@@ -0,0 +1,5 @@
+## Support for reading options from meson.options
+
+Support has been added for reading options from `meson.options` instead of
+`meson_options.txt`. These are equivalent, but not using the `.txt` extension
+for a build file may have some advantages.
```

Comment:
"May  have some" is vague. This should explain at least one advantage out fully.